content/kotlin/concepts/data-types/data-types.md

```diff
@@ -0,0 +1,390 @@
+---
+Title: 'Data-Types'
+Description: 'In Kotlin, a data type is a classification of types of values that can be stored and manipulated in a program.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Primitive Data-Type'
+  - 'Non Primitive Data-Type'
+  
+CatalogContent:
+  - 'learn-Kotlin'
+  - 'paths/computer-science'
+---
+
+A data type is a classification of types of data that determine the possible values and operations that can be performed on that data.
+
+## Type of Data-Types 
```

Comment:
Let's remove this header.

Reply:
Which header you want me to remove ?

Reply:
@dakshdeepHERE: The header "Type of Data Types"

Reply:
@SSwiniarski done 👍🏻